assets/js/add-task.js

```diff
@@ -1,4 +1,4 @@
-import { CLICK_EVENT } from "/assets/js/constants.js";
```

Comment:
it is a convention to put static resources such as images, css stylesheets etc inside `assets` folder. use `scripts` folder for JS files/modules.